applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -79,7 +79,7 @@ Syntax
 
 The ``<cmd>`` command is a string, and can be used as follows:
 
-* ``AT#XCARRIER="app_data"[,<data>][,<instance_id>,<resource_instance_id>]``
+* ``AT#XCARRIER="app_data_set"[,<data>][,<instance_id>,<resource_instance_id>]``
```

Comment:
@MarkusLassila : Could you confirm if the "slm_data_mode.rst" application doc also needs an update here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/slm_data_mode.html#entering-data-mode ?

Reply:
It needs an update. AT#XCARRIER="app_data" -> AT#XCARRIER="app_data_set"